### ISSUE TITLE
Add Dungeon Story Completion for Characters

### ISSUE THREAD

**Issue by Sounomi** [gw2-api/issues#17]
### Feature Description

There's an interest to see which story paths have been completed for a particular character for a dungeon.  This is needed because a character cannot open the explorable paths of a dungeon unless said character has completed the story path already.

Requesting that additional end-point is added to the character data to list which story paths have been completed.  Alternatively, listing which paths for each dungeon have been completed for a character would also be nice.

### Implementation

The structure of the returned data can follow the design of the /accounts/dungeons end-point, merely being a list of which paths have been cleared for that character.

### Example

```json
[
  "ac_story",
  "cm_story",
  "se_story"
]
```

### Anything else?

I seem to remember there being mentioned on Discord that there's already an end-point that provides this functionality but it was never enabled.  If that's the case, perhaps merely enabling that would meet the needs of this request.

**Comment by Archomeda** [comment]
> I seem to remember there being mentioned on Discord that there's already an end-point that provides this functionality but it was never enabled. If that's the case, perhaps merely enabling that would meet the needs of this request.

Correct. https://api.guildwars2.com/v2 lists `/v2/characters/:id/dungeons [d,a]` [for a long time now](https://gist.github.com/darthmaim/16ccbda1944c2d4126f9/revisions?page=2#diff-fb04dcb6970e4c3d1873de51fd5a50d7bb46b3383113602665c350ec40b5f990R52).
